components/Forms/ComponentFactory.tsx

```diff
@@ -65,25 +86,53 @@ export const ComponentFactory: React.VFC<{
   const handleChange = async () => {
     const formData = retrieveFormData()
     if (!formData) return
-    const qs = buildQueryStringFromFormData(formData)
+    const qs = buildQueryStringFromFormData(formData, true)
+
+    const income = formData
+      .get('income')
+      .toString()
+      .replace('$', '')
+      .replace(',', '')
 
-    // client cannot use calculator, their income is too high
-    if (validateIncome(formData.get('income') as string))
-      router.push(`/eligibility?${qs}`)
+    // validate against a client's income and if it's too high, push to the eligibility page with an error
+    if (validateIncome(income)) router.push(`/eligibility?${qs}`)
 
     sendAPIRequest(qs)
   }
 
   return (
-    <form name="ee-form" data-testid="ee-form" action="/eligibility">
+    <form
+      name="ee-form"
+      data-testid="ee-form"
+      action="/eligibility"
+      onSubmit={(e) => e.preventDefault()}
+      noValidate
+    >
+      {/* 
+      <input
+        type="hidden"
+        name="lang"
+        value={useInternationalization('lang')}
+      /> 
+      */}
       {formState.map((field) => {
+        const isChildQuestion =
+          field.category == ('Partner Details' || 'Social Agreement Countries')
+            ? true
+            : false
```

Comment:
Used to add a background for the `'Partner Details' || 'Social Agreement Countries'` categories, Requires changes to the API.

Somewhat related, the order of the questions has changed, so that will need to be updated as well.